bluebrain/repo-bluebrain/packages/neurodamus-model/package.py

```diff
@@ -111,7 +111,10 @@ def build_model(self, spec, prefix):
         # NGV must overwrite other mods, even from the specific
         # models, e.g. ProbAMPANMDA
         if spec.satisfies("+ngv"):
-            copy_all("common_latest/common/mod/ngv", "mod")
+            if spec.satisfies("+metabolism"):
```

Comment:
Conceptually neurodamus-model shouldn't know about a variant of a neocortex.
The other way around is ok (check in the subclass for the +ngv variant). So I wonder if we shouldnt simply make that +ngv and +metabolism cannot be simultanously given? see https://spack.readthedocs.io/en/latest/packaging_guide.html#conditional-variants

Reply:
for the [multiscale_run script](https://bbpgitlab.epfl.ch/molsys/multiscale_run/-/blob/main/multiscale_run_STEPS3.py), we need both +ngv+metabolism. This is because this script calls both dual & triple-run, where dualrun needs +ngv and triplerun needs +metabolism. Any idea in this case?

Reply:
@pramodk what is your view on this, given that the neocortex/mod/metabolism folder is kind of a temporary solution?

Reply:
sorry missed this!

today we can't do `+ngv +metabolism` because there is no guarantee about mod file unique names. So in this PR we can temporarily add conflict with two options. 

In a separate discussion, next week, we can discuss this aspect with Polina and Dan.